jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -1480,13 +1555,17 @@ public Vector3f getScreenCoordinates(Vector3f worldPosition, Vector3f store) {
     }
 
     /**
+     * Determines the display width.
```

Comment:
Should we change this to `Gets` or `Retrieves`.

Reply:
The choice between "Gets" and "Determines" isn't addressed in the style guide.

Reply:
Yes, but determines is a little bit confusing for a getter method, unless this getter is internally changing code.

Reply:
How about "Returns"?

Reply:
I think its better than determines.